libraries/AP_HAL_SITL/AP_HAL_SITL_Namespace.h

```diff
@@ -26,4 +26,5 @@ static const uint8_t SITL_RC_INPUT_CHANNELS = 16;
 // Number of rc output channels
 static const uint8_t SITL_RC_OUTPUT_CHANNELS = 16;
 
+static const uint8_t DEFAULT_RCOUTPUT_FREQ = 50;  // PWM RC : 50Hz
```

Comment:
If this is only used in RCOutput, why is this declared here instead of in the RCOutput class?

You can even remove the initialization of `_freq_hz` in the constructor and make the initialization directly in the declaration in the class.